react/Stack/styles.styl

```diff
@@ -1,20 +1,18 @@
-@require 'settings/breakpoints.styl'
+@require 'utilities/spaces.styl'
 
-.Stack > * + *
-    margin-top .5rem
+spacing_values = {
+    xs: .5rem,
+    s: .75rem,
+    m: 1rem, // the standard, no modifier needed
+    l: 1.5rem,
+    xl: 2rem
+    xxl: 3rem
+}
```

Comment:
If I understand well, this should be removed to use the map from `utilities/spaces.styl`

Reply:
Thanks :) This was indeed the idea, and I mixed up :).